### PR TITLE
GitOps | Test generic-claim workflow

### PR DESCRIPTION
**Pull request by varshab1210**

### CODE REVIEW

**Comment by varshab1210** [comment]
/pj-rehearse pull-ci-redhat-developer-gitops-operator-master-v4.12-kuttl-parallel

File: ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master__v4.12.yaml [hunk 1]
```diff
@@ -79,24 +90,35 @@ tests:
...
           memory: 6Gi
-    workflow: ipi-aws
+    workflow: generic-claim
 - as: kuttl-parallel
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: openshift-ci
```
Comment (2uasimojo) [inline ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master__v4.12.yaml:100]: This should either be `gitops` or `us-east-1`. There is no openshift-ci pool using us-east-2.

**Comment by varshab1210** [comment]
/pj-rehearse pull-ci-redhat-developer-gitops-operator-master-v4.13-kuttl-parallel

**Comment by varshab1210** [comment]
/pj-rehearse pull-ci-redhat-developer-gitops-operator-master-v4.13-kuttl-sequential

**Pull request closed without merging**